src/flowr/internal-session.ts

```diff
@@ -21,8 +21,13 @@ export class FlowrInternalSession {
 		this.shell.tryToInjectHomeLibPath()
 		void this.shell.usedRVersion().then(version => {
 			if(version == null){
-				// this will soon be a lot more useful because it'll ask if we want to install R
-				void vscode.window.showErrorMessage('R was not found on your system. Is it installed correctly?')
+				const seeDoc = 'See documentation'
+				void vscode.window.showErrorMessage('The R version could not be determined. R needs to be installed and part of your PATH environment variable.', seeDoc)
+					.then(s => {
+						if(s === seeDoc){
+							void vscode.env.openExternal(vscode.Uri.parse('https://github.com/Code-Inspect/vscode-flowr/blob/main/README.md#using'))
+						}
+					})
 			} else {
```

Comment:
By retrieving the shell we could guard 
< 3 i do not think that we work
3 <= x < 4 ~> at least not tested. Some things do probably work.

4 <= x ~> tested, no warning needed